__tests__/app.test.js

```diff
@@ -430,3 +430,121 @@ test("400: bad request message sent for invalid comment_id e.g. string", () => {
     .expect(400)
     .then(({ body }) => expect(body.msg).toBe("bad request"));
 });
+describe("GET /api", () => {
+  test("200: responds with the endpoints JSON explaining the functioning of the API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
```

Comment:
I probably wouldnt test this that much, i'd just check for an object. 
At the moment this test is really brittle and will break everytime you add something to the doc